project/rroykuo/README.md

```diff
@@ -0,0 +1,52 @@
+# 3D-Retro-Bouncing-DVD
+
+## Basic Information
+Bouncing DVD is a retro screensaver that would appear in people's childhood. When taking tour bus and watching movie via DVD, we could see the screensaver due to device idling for a while. My object is to reproduce a DVD screensaver but in 3D space. Moreover, I'll try to make the interface to allow users create new DVD plate at any position.
+
+![](https://i.imgur.com/EYZdwQy.gif)
+
+## Problem To Solve
+
+First, we might simply this project into a collision simulator. However, balls should be transformed into a plate (words of DVD). The original physical functions need to consider non-uniform surface.
+
+For user interface, I'll use pygame to utilize a window show the 3D space. Users could use mouse to drag and click to generate a new DVD plate. The library of pygame is intent to render 2D object, so PyOpenGL would specify 3D object in this project. Therefore, I must figure out what value or how many variable would be sent to C++ defined function. If there are extremendous things to calculate, the system will not handle.
```

Comment:
You may also consider to directly write code in OpenGL or a toolkit like Qt (3D).